jenkins/pipelines/cd/tiflow-docker.groovy

```diff
@@ -0,0 +1,154 @@
+def podYaml = '''
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dinp
+  namespace: jenkins-cd
+spec:
+  containers:
+  - name: docker
+    image: docker:dind
+    args: ["--registry-mirror=https://registry-mirror.pingcap.net"]
+    env:
+    - name: REGISTRY
+      value: hub.pingcap.net
+    - name: DOCKER_TLS_CERTDIR
+      value: ""
+    - name: DOCKER_HOST
+      value: tcp://localhost:2375
+    securityContext:
+      privileged: true
+    tty: true
+    readinessProbe:
+      exec:
+        command: ["docker", "info"]
+      initialDelaySeconds: 10
+      failureThreshold: 6
+  - name: jnlp
+    image: jenkins/inbound-agent:4.10-3
+'''
+
+def ImageTag = params.REVISION
+
+pipeline {
+    agent none
+    triggers {
+        cron('@daily')
+    }
+    parameters {
+        string(name: 'REVISION', defaultValue: 'master', description: 'branch or tag or hash, master will push to latest')
+    }
+    stages {
+        stage ("parallel build docker by arch") {
+            parallel{
+                stage("amd64"){
+                    agent {
+                        kubernetes {
+                            yaml podYaml
+                            defaultContainer 'docker'
+                        }
+                    }
+                    stages{
+                        stage('configure docker'){
+                            environment {HUB = credentials('harbor-pingcap')}
+                            steps{
+                                sh 'printenv HUB_PSW | docker login hub.pingcap.net -u ${HUB_USR} --password-stdin'
+                            }
+                        }
+                        stage("checkout"){
+                            steps{
+                                checkout scm: [$class: 'GitSCM',
+                                               branches: [[name: params.REVISION]],
+                                               extensions: [[$class: 'LocalBranch']],
+                                               userRemoteConfigs: [[credentialsId: 'github-sre-bot-ssh', url: 'git@github.com:pingcap/tiflow.git']]]
```

Comment:
env.ImageTag = sh(returnStdout: true, script: "git rev-parse HEAD").trim()

Need to override image tag with git hash.